spec/models/category_promo_spec.rb

```diff
@@ -0,0 +1,5 @@
+describe CategoryPromo do
+  it 'associations' do
```

Comment:
we've got this in sentence format [here](https://github.com/moneyadviceservice/cms/pull/227/files#diff-5cfe7e4f7bab0f6559c519ec3aa95b18R29), but you could equally use `specify 'associations' do`